pkg/storage/client_test.go

```diff
@@ -841,6 +841,18 @@ func (m *multiTestContext) restartStore(i int) {
 	m.senders[i].AddStore(m.stores[i])
 }
 
+// disableCircuitBreakersToStore ensures that no store's RaftTransport will
+// reject requests just because past connections to the given store have failed.
+// If you're stopping store and restarting them, you may want to use this to
+// ensure that errors from when the store was down don't cause other nodes to
+// avoid opening new connections to it.
+func (m *multiTestContext) disableCircuitBreakersToStore(i int) {
+	nodeID := roachpb.NodeID(i + 1)
+	for _, transport := range m.transports {
+		transport.GetCircuitBreaker(nodeID).ShouldTrip = nil
+	}
+}
```

Comment:
Ah, yeah. That's basically what I was wondering about in my PR description. It'd be a little clunky to do it like this, though, since we'd have to do it for the cross-product of nodes, and some nodes can be added after the multiTestContext is created.

We'd want to change the way the breakers are constructed. I was thinking that may involve a fair amount of work so I was just proposing it rather than doing it, but now I see that we already have a BreakerFactory in place :) Switched over to always disabling the breakers. I've been running `make stress` on the storage package for about five minutes now after making this change and still haven't run into any failures, so it looks like this may not have broken any tests' assumptions, which is a pleasant surprise.